test/remote/gateways/remote_stripe_test.rb

```diff
@@ -43,6 +48,25 @@ def test_successful_purchase
     assert response.params["paid"]
     assert_equal "ActiveMerchant Test Purchase", response.params["description"]
     assert_equal "wow@example.com", response.params["metadata"]["email"]
+  end
+
+  # this actually shouldn't be legal
```

Comment:
Because Stripe technically has everything they need to capture funds, but it's ill-advised to do in actual practice, since we need a message back to the chip to actually finalize the transaction on the card level. I put the comment there so we know to never use a `purchase` in the EMV case.